Jena version mismatch

We're using Jena 3.9.0 to support `trellis-rdfa` but Commons RDF is still stuck on 3.5.0. I've created [a Commons ticket](https://issues.apache.org/jira/browse/COMMONSRDF-80) for the long-term fix, but for now, trying to boot Trellis is giving me:
```
2018-12-29 11:41:29,742 ERROR [stderr] (main)     Caused by: java.util.ServiceConfigurationError: org.apache.commons.rdf.api.RDF: Provider org.apache.commons.rdf.jena.JenaRDF could not be instantiated
2018-12-29 11:41:29,742 ERROR [stderr] (main)     Caused by: java.util.ServiceConfigurationError: org.apache.jena.sys.JenaSubsystemLifecycle: Provider org.apache.jena.riot.system.InitRIOT not a subtype"}}
```
which points to the version mismatch.

--- a/build.gradle
+++ b/build.gradle
@@ -149,9 +149,7 @@ allprojects { subproj ->
     }
 
     configurations.all {
-        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
         resolutionStrategy.dependencySubstitution {
-            substitute module("org.apache.jena:jena-osgi") with module("org.apache.jena:jena-arq:$jenaVersion")
             substitute module("org.apache.geronimo.specs:geronimo-annotation_1.2_spec") with module ("javax.annotation:javax.annotation-api:$javaxAnnotationsVersion")
         }
         resolutionStrategy.eachDependency { DependencyResolveDetails details ->

--- a/components/app-triplestore/build.gradle
+++ b/components/app-triplestore/build.gradle
@@ -15,11 +15,15 @@ dependencies {
     compile("javax.activation:javax.activation-api:$activationApiVersion")
     compile("javax.jms:javax.jms-api:$jmsApiVersion")
     compile("org.apache.activemq:activemq-client:$activeMqVersion")
-    compile("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    compile("org.apache.jena:jena-arq:$jenaVersion")
     compile("org.apache.jena:jena-rdfconnection:$jenaVersion")
     compile("org.apache.jena:jena-tdb2:$jenaVersion")
     compile("org.apache.tamaya:tamaya-core:$tamayaVersion")
     compile("org.slf4j:slf4j-api:$slf4jVersion")
+    compile("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
 
     compile project(':trellis-constraint-rules')
     compile project(':trellis-io-jena')

--- a/components/app/build.gradle
+++ b/components/app/build.gradle
@@ -25,7 +25,11 @@ dependencies {
     testImplementation("ch.qos.logback:logback-classic:$logbackVersion")
     testImplementation("io.dropwizard:dropwizard-client:$dropwizardVersion")
     testImplementation("io.dropwizard:dropwizard-testing:$dropwizardVersion")
-    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
+    testImplementation("org.apache.jena:jena-arq:$jenaVersion")
     testImplementation("org.apache.jena:jena-rdfconnection:$jenaVersion")
     testImplementation("org.apache.tamaya:tamaya-core:$tamayaVersion")
     testImplementation("org.awaitility:awaitility:$awaitilityVersion")

--- a/components/constraint-rules/build.gradle
+++ b/components/constraint-rules/build.gradle
@@ -12,7 +12,11 @@ dependencies {
 
     implementation project(':trellis-vocabulary')
 
-    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
+    testImplementation("org.apache.jena:jena-arq:$jenaVersion")
     testImplementation("org.mockito:mockito-core:$mockitoVersion")
 }
 

--- a/components/file/build.gradle
+++ b/components/file/build.gradle
@@ -13,7 +13,11 @@ dependencies {
 
     implementation("commons-codec:commons-codec:$commonsCodecVersion")
     implementation("commons-io:commons-io:$commonsIoVersion")
-    implementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    implementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
+    implementation("org.apache.jena:jena-arq:$jenaVersion")
     implementation("org.apache.tamaya:tamaya-api:$tamayaVersion")
     implementation("org.slf4j:slf4j-api:$slf4jVersion")
     implementation project(':trellis-vocabulary')

--- a/components/io-jena/build.gradle
+++ b/components/io-jena/build.gradle
@@ -12,7 +12,11 @@ dependencies {
     api project(':trellis-api')
 
     implementation("commons-io:commons-io:$commonsIoVersion")
-    implementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    implementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
+    implementation("org.apache.jena:jena-arq:$jenaVersion")
     implementation("org.apache.tamaya:tamaya-api:$tamayaVersion")
     implementation("org.slf4j:slf4j-api:$slf4jVersion")
     implementation project(':trellis-vocabulary')

--- a/components/rdfa/build.gradle
+++ b/components/rdfa/build.gradle
@@ -16,7 +16,11 @@ dependencies {
     implementation("org.apache.tamaya:tamaya-api:$tamayaVersion")
     implementation project(':trellis-vocabulary')
 
-    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
+    testImplementation("org.apache.jena:jena-arq:$jenaVersion")
     testImplementation("org.apache.tamaya:tamaya-core:$tamayaVersion")
     testImplementation("org.mockito:mockito-core:$mockitoVersion")
 }

--- a/components/test/build.gradle
+++ b/components/test/build.gradle
@@ -12,7 +12,11 @@ dependencies {
     implementation("io.jsonwebtoken:jjwt-api:$jjwtVersion")
     implementation("javax.ws.rs:javax.ws.rs-api:$jaxrsVersion")
     implementation("org.apache.commons:commons-text:$commonsTextVersion")
-    implementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    implementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: "org.apache.jena", module: "jena-osgi"
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
+    implementation("org.apache.jena:jena-arq:$jenaVersion")
     implementation("org.awaitility:awaitility:$awaitilityVersion")
     implementation("org.junit.jupiter:junit-jupiter-api:$junitVersion")
     implementation project(':trellis-api')

--- a/components/triplestore/build.gradle
+++ b/components/triplestore/build.gradle
@@ -12,9 +12,13 @@ dependencies {
     api project(':trellis-api')
     api project(':trellis-audit');
 
-    implementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    implementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
     implementation("org.apache.jena:jena-rdfconnection:$jenaVersion")
     implementation("org.apache.jena:jena-tdb2:$jenaVersion")
+    implementation("org.apache.jena:jena-arq:$jenaVersion")
     implementation("org.apache.tamaya:tamaya-api:$tamayaVersion")
     implementation("org.slf4j:slf4j-api:$slf4jVersion")
     implementation project(':trellis-vocabulary')

--- a/components/webac/build.gradle
+++ b/components/webac/build.gradle
@@ -17,7 +17,11 @@ dependencies {
     implementation project(':trellis-vocabulary')
 
     testImplementation("ch.qos.logback:logback-classic:$logbackVersion")
-    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
+    testImplementation("org.apache.jena:jena-arq:$jenaVersion")
     testImplementation("org.apache.tamaya:tamaya-core:$tamayaVersion")
     testImplementation("org.mockito:mockito-core:$mockitoVersion")
     testImplementation project(':trellis-triplestore')

--- a/core/api/build.gradle
+++ b/core/api/build.gradle
@@ -13,7 +13,11 @@ dependencies {
 
     testImplementation("commons-io:commons-io:$commonsIoVersion")
     testImplementation("org.mockito:mockito-core:$mockitoVersion")
-    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion");
+    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
+    testImplementation("org.apache.jena:jena-arq:$jenaVersion")
     testImplementation("org.apache.commons:commons-text:$commonsTextVersion")
     testImplementation("org.slf4j:slf4j-api:$slf4jVersion")
     testImplementation project(':trellis-vocabulary')

--- a/core/vocabulary/build.gradle
+++ b/core/vocabulary/build.gradle
@@ -10,7 +10,11 @@ ext {
 dependencies {
     api("org.apache.commons:commons-rdf-api:$commonsRdfVersion")
 
-    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion")
+    testImplementation("org.apache.commons:commons-rdf-jena:$commonsRdfVersion") {
+        exclude group: 'org.apache.jena', module: 'jena-osgi'
+        exclude group: 'org.apache.servicemix.bundles', module: 'org.apache.servicemix.bundles.xerces'
+    }
+    testImplementation("org.apache.jena:jena-arq:$jenaVersion")
     testImplementation("org.mockito:mockito-core:$mockitoVersion")
 
     testRuntimeClasspath("ch.qos.logback:logback-classic:$logbackVersion")

